### ISSUE TITLE
TBCSessionProxy: fix integration test

### ISSUE THREAD

**Issue by meox** [extendi/sunspot#13]

**Comment by meox** [comment]
```
Failures:
  1) Sunspot::SessionProxy::TbcSessionProxy remove remove_by_id
     Failure/Error: expect(Post.search.total).to eq(Post.count - 1)
     
       expected: 9
            got: 10
     
       (compared using ==)
     # ./spec/tbc_proxy_spec.rb:226:in `block (3 levels) in <top (required)>'
     # /home/travis/.rvm/gems/ruby-2.5.1/gems/bundler-1.17.3/lib/bundler/cli/exec.rb:74:in `load'
     # /home/travis/.rvm/gems/ruby-2.5.1/gems/bundler-1.17.3/lib/bundler/cli/exec.rb:74:in `kernel_load'
     # /home/travis/.rvm/gems/ruby-2.5.1/gems/bundler-1.17.3/lib/bundler/cli/exec.rb:28:in `run'
     # /home/travis/.rvm/gems/ruby-2.5.1/gems/bundler-1.17.3/lib/bundler/cli.rb:463:in `exec'
     # /home/travis/.rvm/gems/ruby-2.5.1/gems/bundler-1.17.3/lib/bundler/cli.rb:27:in `dispatch'
     # /home/travis/.rvm/gems/ruby-2.5.1/gems/bundler-1.17.3/lib/bundler/cli.rb:18:in `start'
     # /home/travis/.rvm/gems/ruby-2.5.1/gems/bundler-1.17.3/lib/bundler/friendly_errors.rb:124:in `with_friendly_errors'
```

**Comment by meox** [comment]
Will be fixed in this PR: Solr repair #12
